server.go

```diff
@@ -182,10 +188,24 @@ func RPCDecompressor(dc Decompressor) ServerOption {
 }
 
 // MaxMsgSize returns a ServerOption to set the max message size in bytes for inbound mesages.
-// If this is not set, gRPC uses the default 4MB.
+// If this is not set, gRPC uses the default limit. Deprecated: use MaxReceiveMessageSize instead.
 func MaxMsgSize(m int) ServerOption {
+	return MaxReceiveMessageSize(m)
+}
+
+// MaxReceiveMessageSize returns a ServerOption to set the max message size in bytes for inbound mesages.
```

Comment:
The comment,
`... max message size in bytes the server can receive.`, since we name the option `...Receive...`.